roles/nuage_master/vars/main.yaml

```diff
@@ -29,6 +29,6 @@ nuage_service_account_config:
 nuage_tasks:
     - policy add-cluster-role-to-user cluster-reader {{ nuage_service_account }} 
 
-nuage_master_cspadminpasswd: ''
-nuage_master_adminusername: 'admin'
-nuage_master_adminuserpasswd: 'admin'
+nuage_master_csp_admin_passwd: "{{nuage_master_cspadminpasswd | default('') }}"
+nuage_master_admin_username: "{{nuage_master_adminusername | default('admin') }}"
+nuage_master_admin_userpasswd: "{{nuage_master_adminuserpasswd | default('admin') }}"
```

Comment:
Man, I need to stop commenting from my phone...  The correct location is `roles/nuage_master/defaults/main.yaml`